src/examples/basicRX.cpp

```diff
@@ -63,21 +100,43 @@ int main(int argc, char** argv)
     device->SetMessageLogCallback(LogCallback);
     device->Init();
 
+    const auto& chipDescriptor = device->GetDescriptor().rfSOC[chipIndex];
+
+    // Default RX path when none is provided via the command lime.
+    //
+    // TODO: Choose antenna which makes sense for the device.
+    // For example, rxPath=2 is LNAL_NC for LimeSDR Mini, which will result in
+    // effectively no signal in the received samples.
+    int rxPath = 2;
+
+    const std::string rxAntennaName = args::get(rxpathFlag);
+    if (!rxAntennaName.empty())
+    {
+        rxPath = AntennaNameToIndex(chipDescriptor.pathNames.at(TRXDir::Rx), rxAntennaName);
+        if (rxPath < 0)
+        {
+            DeviceRegistry::freeDevice(device);
+            return EXIT_FAILURE;
+        }
+    }
+
+    std::cout << "Using antenna "sv << chipDescriptor.pathNames.at(TRXDir::Rx).at(rxPath) << std::endl;
+
     // RF parameters
     SDRConfig config;
     config.channel[0].rx.enabled = true;
     config.channel[0].rx.centerFrequency = frequencyLO;
     config.channel[0].rx.sampleRate = sampleRate;
     config.channel[0].rx.oversample = 2;
     config.channel[0].rx.lpf = 0;
-    config.channel[0].rx.path = 2; // TODO: replace with string names
+    config.channel[0].rx.path = rxPath;
     config.channel[0].rx.calibrate = false;
     config.channel[0].rx.testSignal.enabled = false;
 
     config.channel[0].tx.enabled = false;
     config.channel[0].tx.sampleRate = sampleRate;
     config.channel[0].tx.oversample = 2;
-    config.channel[0].tx.path = 2; // TODO: replace with string names
+    config.channel[0].tx.path = rxPath;
```

Comment:
```suggestion
    config.channel[0].tx.path = 0;
```
in general case assigning index of rxPath value to Tx path might not always be valid. Since this is just Rx example, Tx path can be set to 0.